Core/include/Acts/Propagator/EigenStepper.hpp

```diff
@@ -73,22 +72,24 @@ class EigenStepper {
     /// @param [in] stolerance is the stepping tolerance
     ///
     /// @note the covariance matrix is copied when needed
-    template <typename parameters_t>
+    template <typename parameters_t,
+              std::enable_if_t<parameters_t::is_local_representation, int> = 0>
```

Comment:
When I added these variables to the parameters, the naming pattern for variables weren't developed yet. I think it has to be something like `s_isBoundParameters` or so. @msmk0 could help suggesting names.

Reply:
Good call ...

Reply:
I renamed it to s_boundRepresentation

Reply:
Dropping this following @msmk0 's suggestion.